src/components/10-atoms/radio/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.1.6
+
+- Fix: Iconified radio buttons no longer have the attribute value `fill="currentColor"`. (#1942)
```

Comment:
This is a possible breaking change

Reply:
It is a fix because it was wrong before. A major change in the icons introduced this behavior and this bug is fixed with this version.